src/libcore/num/f64.rs

```diff
@@ -8,7 +8,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Operations and constants for 64-bits floats (`f64` type)
+//! Operations and constants for 64-bits floats.
+//!
+//! *[See also the `f64` primitive type](../../std/primitive.f64.html).*
```

Comment:
.... basically, all of the primitive stuff that's in `libcore`. We shouldn't change these.

All of the docs for primitive types are in the `primitive_docs` submodule of `std`.